actionpack/lib/action_dispatch/http/mime_negotiation.rb

```diff
@@ -139,6 +141,31 @@ def negotiate_mime(order)
         order.include?(Mime::ALL) ? format : nil
       end
 
+      class VariantInquirer # :nodoc:
+        delegate :==, :each, :empty?, to: :@variants
```

Comment:
I still don't understand why we need #==. The point of this is to provide a better interface, i.e. #any?.

Are there places in the framework that use == with variants?